pkg/crc/oc/clusteroperator.go

```diff
@@ -0,0 +1,79 @@
+package oc
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	crcos "github.com/code-ready/crc/pkg/os"
+	"time"
+)
+
+var ignoreClusterOperators = []string{"monitoring", "machine-config", "marketplace"}
+
+type ClusterOperator struct {
+	Items []struct {
+		Metadata struct {
+			Name string `json:"name"`
+		} `json:"metadata"`
+		Status struct {
+			Conditions []struct {
+				LastTransitionTime time.Time `json:"lastTransitionTime"`
+				Reason             string    `json:"reason"`
+				Status             string    `json:"status"`
+				Type               string    `json:"type"`
+			} `json:"conditions"`
+		} `json:"status,omitempty"`
+	} `json:"items"`
+}
+
+func GetClusterOperatorStatus(ocPath string, kubeconfigPath string) (bool, error) {
+	allAvailable := true
+	data, stderr, err := crcos.RunWithDefaultLocale(ocPath, "--kubeconfig", kubeconfigPath, "get", "co", "-ojson")
+	if err != nil {
+		return false, fmt.Errorf("%s - %v", stderr, err)
+	}
+
+	var co ClusterOperator
+
+	err = json.Unmarshal([]byte(data), &co)
+	if err != nil {
+		return false, err
+	}
+	for _, c := range co.Items {
+		if isContain(c.Metadata.Name, ignoreClusterOperators) {
+			continue
+		}
+		for _, con := range c.Status.Conditions {
+			switch con.Type {
+			case "Available":
+				if con.Status != "True" {
+					logging.InfoF(c.Metadata.Name, "operator not available, retrying ...")
+					logging.DebugF(c.Metadata.Name, "operator not available, Reason: ", con.Reason)
+					allAvailable = false
+				}
+			case "Degraded":
+				if con.Status != "False" {
+					logging.InfoF(c.Metadata.Name, "operator is degraded, retrying ...")
```

Comment:
Same here.. `logging.InfoF("%s operator is degraded, retrying ...", c.Metadata.Name)`

Reply:
I used `Info` only instead `InfoF` now.